test/Passes/pass-conv-to-matmul.mlir

```diff
@@ -1,4 +1,8 @@
-// RUN: tpp-opt %s -rewrite-conv-to-matmul-or-brgemm -canonicalize -split-input-file | FileCheck %s
+// RUN: tpp-opt %s -rewrite-conv-to-matmul-or-brgemm -canonicalize -split-input-file | FileCheck -check-prefix=UNBLCKHWCF %s
```

Comment:
I'm guessing each RUN line matches only one of the tests, so it would be good to check that the other two are not matched and still have the original operation.

Reply:
We map both blocked and non-blocked convs. So the first run also map to linalg even if it does not block the conv.

Reply:
Would be nice to have a CHECK line for each run on each example, to make explicit what we're expecting here.

Reply:
Made some changes. I just want to check that we are able to map a blocked conv to matmul. I updated the test this way.

Reply:
As it is, this are three separate tests. Even though the other RUN lines pass on the other (non-CHECKed) tests, we have no way of knowing if their execution is correct, as there are no CHECK lines. Keeping them like this gives the wrong impression that we know it works.

My point is simple: either add all CHECKs to all splits, or split this into three separate tests.

Reply:
I understand your point. There are not more three lines of RUN, just a single one.

Reply:
I'm sorry, Github showed me the old file still. What a crap interface!